.github/workflows/test-database.yml

```diff
@@ -70,6 +70,11 @@ jobs:
           docker compose --profile mssql --profile postgres --profile mysql up -d
         shell: pwsh
 
+      - name: Add Maria DB
```

Comment:
```suggestion
      - name: Add MariaDB for Unified
```

Reply:
I've always been told it's supposed to be lowercase... but will change it.